<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountStatement</class>
 <widget class="QDialog" name="accountStatement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>虎图帐</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/icon.jpg</normaloff>:/img/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#chart{
	background-color: #fafafa;
}
QLabel {
	font-family: &quot;微软雅黑&quot;;
	font-size: 18px;
	color: #666;
}

QPushButton#AccountStatementBtn,QPushButton#AccountingBtn,QPushButton#ChartBtn,QPushButton#HomepageBtn {
    color: #fff;
    font-size: 45px;
    font-family: &quot;逐浪雅宋体&quot;;
    line-height: 80px;
    height: 80px;
    border: 0;
}

QPushButton#AccountStatementBtn:hover,QPushButton#AccountingBtn:hover,QPushButton#ChartBtn:hover,QPushButton#HomepageBtn:hover {

	background-image: url(:/img/hoverBackground.png);
}
QPushButton#AccountStatementBtn:pressed,QPushButton#AccountingBtn:pressed,QPushButton#ChartBtn:pressed,QPushButton#HomepageBtn:pressed {

	background-image: url(:/img/pressedBackground.png);
}
QWidget#leftContainer{
    background-color:#ffbc2e
}
QComboBox {
	line-height: 20px;
    height: 20px;
    font-family: &quot;微软雅黑&quot;;
    font-size: 18px;
    color: #666;
	border: 1px solid #ccc;
}

QComboBox::down-arrow {
	width: 16px;
  
	image: url(:/img/btnDownUnChecked.png);
	background-color: #fff;
}
QComboBox::down-arrow:pressed{
    
	image: url(:/img/btnDownChecked.png);
}
QComboBox::drop-down {
	border: 0;
}
#EditBtn {
	border: 0;
	font-family: &quot;逐浪雅宋体&quot;;
	color: #fff;
	font-size: 27px;
	background-color: #ffbc2e;
	border-radius: 20px;
}
#setBtn {
	border: 0;
	background-image: url(:/img/keyUnclicked.png);
}
#setBtn:pressed {
	background-image :url(:/img/keyCLicked.png)
}</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>671</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="IncomeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="InOutcomeBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>收入与支出</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>收入</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>支出</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="IncomeLabel_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日支出：</string>
   </property>
  </widget>
  <widget class="QLabel" name="OutcomeLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="OutcomeLabel_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日收入：</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lOuterLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="leftContainer" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>501</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="leftLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="HomepageBtn">
          <property name="text">
           <string>首页</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AccountingBtn">
          <property name="text">
           <string>记账</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AccountStatementBtn">
          <property name="text">
           <string>流水</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChartBtn">
          <property name="text">
           <string>图析</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="EditBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>460</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QComboBox" name="YearBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2025</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2026</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2027</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2028</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2029</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2030</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="MonthBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>7</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="DayBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>12</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="timeLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>159</width>
     <height>53</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>97</width>
     <height>56</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
